### ISSUE TITLE
[ACTIVE_STALE_TAKEOVER] SWARM-1759 — PR910 EDRNPA real-execution convergence

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1759]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1759
STATUS: ACTIVE_STALE_TAKEOVER_PENDING_POSTCLAIM_DIRECT_ARBITRATION
SWARM_LANE_KEY: D03|PR910|EDRNPA|CURRENT-SOURCE-REAL-EXECUTION-CONVERGENCE-V1
PREFERRED_LANE: #6 D05
PREFERRED_WORK_KIND: DATA_SOURCE_OR_PIPELINE
ROUTING_DECISION: rotate preferred D05 lane to incumbent D03 PR910 because D05/recovery surfaces are already owned and current dominant learned-20M blocker is lawful corpus capacity; preferred work-kind matches exactly.
PARENT_ISSUES: #4 #15 #548 #723 #910 #1048 #1186 #1355
BASE_SHA: 2da11c62f066671a1122888ec116242502f4fb9c
INCUMBENT_PR: #910
INCUMBENT_BRANCH: d03/edrnpa-open-data-probe-20260907
INCUMBENT_HEAD: e61a59a30ba76cff86d98030ab996ca21f32e6fb
PREDECESSOR_OWNER: #1355
PREDECESSOR_CREATED_AT_UTC: 2026-09-11T21:59:08Z
PREDECESSOR_LAST_UPDATE_UTC: 2026-09-11T22:09:21Z
PREDECESSOR_DEFAULT_LEASE_EXPIRED_UTC: 2026-09-12T03:59:08Z
CLAIMED_AT_UTC: THIS_ISSUE_UPDATE_TIMESTAMP_IS_AUTHORITY
LEASE_UNTIL_UTC: six hours from this claim update unless explicitly released sooner

STALE-TAKEOVER EVIDENCE:
- #1355 won its arbitration but then made no later durable Product checkpoint after 2026-09-11T22:09:21Z; its 6h lease is expired by multiple days.
- PR910 remains OPEN/non-draft/mergeable at exact e61a59a30ba76cff86d98030ab996ca21f32e6fb and unchanged since 2026-09-11T04:28:58Z.
- branch-specific current Actions checks show 0 queued and 0 in-progress runs, so no active CI is plausibly owned by #1355.
- fresh exact-semantic open search before this claim found #1355 as the newest owner; no successor exact-semantic claim was found.
- current main remains exact 2da11c62f066671a1122888ec116242502f4fb9c immediately before claim.

PHYSICAL NEGATIVE EVIDENCE TO CONSUME, NOT REPEAT:
- exact head e61a59a... temp run 34562359348 / job 103147530592 succeeded for focused diagnostics and real official download.
- observed current source: 611,865,397 bytes; MD5 0ea96e1582e5584ced79be1027f0ae55; SHA-256 a87b23eff3aadd2ff2dbed24cdeaaaf37862ecebe65b5f1edad02d5a95cde431; nested XML edrnpa_texts_2026-09-06.xml, 3,382,489,701 bytes.
- real diagnosis: 144,107 documents, all with some extracted text but all too short under current extraction; only 1,471,980 normalized bytes total; `par` elements were absent while observed attribute name `def` exposed the parser/schema mismatch. This is negative execution evidence, NOT a realized 4 MB candidate.
- current Product code is already rebound to catalog update 2026-09-08T15:02:00+03:00 and MD5 0ea96e1582e5584ced79be1027f0ae55, but extraction still assumes the wrong text shape.

OBJECTIVE:
Finish the existing PR910 lineage end-to-end: repair extraction for the physically observed current XML shape without loading the 3.38 GB XML in memory; preserve bounded/fail-closed archive/XML handling, quality/privacy quarantine and deterministic selection; validate with focused/adversarial tests; perform genuine two-clean LOCAL_FREE acquisitions/materializations through the existing execution carrier; retain compact text-free identities; remove the temporary workflow from final Product head; obtain exact-head shared CI; hand to fresh independent audit. Do not create a second parser/framework/PR.

OWNED_SURFACES:
- incumbent PR910 branch and its three changed files only: tools/probe_d03_edrnpa_open_data_v1.py, tests/test_probe_d03_edrnpa_open_data_v1.py, .github/workflows/d03-edrnpa-real-execution-temp.yml;
- observed EDRNPA schema adaptation, bounded streaming extraction, deterministic zero-credit materialization and exact text-free execution evidence;
- temporary specialist workflow only while needed for substantive real execution, then delete it before final shared CI.

AVOID_SURFACES:
- no central global dedup/decontamination/G05/G06/balance/split/packing/tokenizer/readiness/D05/D06/integration mutation;
- no PR945/PR1459/PDR occupied semantics;
- no tokenizer fit, optimizer/model update, final-test outcome read, paid compute, foreign pretrained weights, external LLM/API corpus intelligence;
- source materialization alone grants zero canonical/training/unique-loss/exposure credit.

PACKAGE_DIMENSIONS:
1. implementation_or_primary_research: repair the real observed XML extraction shape on incumbent code;
2. focused_tests: preserve/run exact source parser tests;
3. adversarial_or_negative_tests: malformed/unsafe XML/ZIP, schema drift, extraction-boundary, deterministic zero-credit tests;
4. live_authority_binding: exact current catalog/resource MD5/SHA and physical real-run evidence;
5. end_to_end_or_integration_proof: two independent LOCAL_FREE acquisitions/materializations with byte-identical text-free report/inventory identities, or exact truthful negative evidence;
6. documentation_or_operator_handoff: durable issue/PR execution receipt and independent-audit handoff.

ACCEPTANCE:
- mandatory postclaim direct OPEN issue and OPEN PR collections through exact empty sentinels show no earlier currently-live exact-semantic owner; if collision exists, abandon/rotate before Product write;
- extraction consumes the real observed source structure under streaming memory bounds and keeps DTD/entities, path traversal, symlink, encryption, decompression/resource bombs fail-closed;
- current immutable source identity is checked before candidate materialization and source drift fails closed;
- two clean executions either prove deterministic candidate/report identities with exact count/bytes or retain exact non-fabricated negative evidence;
- every emitted record stays training_eligible=false and evaluation_eligible=false; rights/provenance and all downstream corpus gates remain mandatory;
- final Product head contains no specialist workflow; focused tests and shared exact-head CI terminal SUCCESS;
- fresh different-worker audit required; producer does not self-certify or merge.

FALLBACK:
If arbitration loses or the current source cannot lawfully/materially yield useful text after a bounded schema repair, release PR910 immediately and rotate this same worker issue to the next highest-yield unowned UA source/execution lane; do not burn the turn on repeated retries.

TRUTH_BOUNDARY_AT_CLAIM:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
POSTCLAIM_DIRECT_ARBITRATION: WIN for exact key `D03|PR910|EDRNPA|CURRENT-SOURCE-REAL-EXECUTION-CONVERGENCE-V1`.

Evidence:
- direct OPEN issue collection was enumerated newest-first at 100/page through exact empty sentinel: pages 1..12 non-empty, page 13 = `[]`;
- separate direct OPEN PR collection was enumerated newest-first at 100/page through exact empty sentinel: pages 1..5 non-empty, page 6 = `[]`;
- race refresh of newest OPEN page exposed later same-key claimant #1763, created 2026-09-14T00:08:30Z;
- SWARM-1759 was created 2026-09-14T00:05:21Z and published the same exact key at updated_at 2026-09-14T00:06:42Z, after predecessor #1355's lease had expired by multiple days;
- therefore under SWARM-300-V2 exact-key arbitration (earliest valid post-expiry claimant; then lower issue number), #1759 is the winner and #1763 is the later duplicate claimant;
- older #1355/#1186/#1048 are expired predecessor semantics; no earlier still-live successor was found.

PRODUCT_OWNERSHIP: SWARM-1759 owns incumbent PR910 same-lineage Product mutation for this lease. #1763 must yield/rotate before Product mutation. I will late-bind current main, PR910 head/state, exact-key owners and CI pressure again immediately before Product write.

Scientific truth unchanged: optimized-target exposure=0; tokenizer fit unauthorized; optimizer updates/training/learned weights/final-test outcomes/paid compute/foreign pretrained weights/external LLM corpus intelligence remain false/zero.

**Comment by Oleksii-debug** [comment]
PHYSICAL_SCHEMA_DIAGNOSIS CHECKPOINT — exact branch head `59c333a24a9775ee0a224887d480633f47cd8860`, run `34791932348`, job `103817653377` terminal SUCCESS.

Focused regressions: 18 passed; py_compile passed.

Payload-free aggregate physical result from the exact current official source:
- source = 611,865,397 bytes; MD5 `0ea96e1582e5584ced79be1027f0ae55`; SHA-256 `a87b23eff3aadd2ff2dbed24cdeaaaf37862ecebe65b5f1edad02d5a95cde431`;
- nested XML = `edrnpa_texts_2026-09-06.xml`, 3,382,489,701 bytes;
- exact observed target paths: `rna/database/document/item` 288,214; `rna/database/document/item/text` 144,107; `rna/database/document/item/richtext` 144,107; `rna/database/document/item/richtext/par` 562,490;
- `par` attributes are exactly `def`; 562,490/562,490 have it, aggregate `def` UTF-8 bytes=562,490 and max=1 byte — marker/metadata, not legal payload;
- `par` itertext = 3,350,129,627 UTF-8 bytes across 562,490 elements, 562,429 nonempty;
- sibling `text` itertext = only 1,760,194 bytes across 144,107 elements;
- one XML10-forbidden control byte removed with identity `d8d16dc7be636a2571527b888f8cfd8c5549c0ce2ae9b748f17f7d391bbf44c4`;
- no payload text or payload attribute values printed.

ROOT CAUSE CONFIRMED: incumbent parser required `par` to be under a `text` ancestor and therefore skipped the real legal text, then consumed the short sibling `<item><text>`. The real legal payload is the exact schema path `rna/database/document/item/richtext/par`.

Official NAIS metadata independently describes this XML/ZIP dataset as containing legal-act text and publishes an XSD structure link, so the physical observation is semantically consistent with the publisher contract.

NEXT SAME-LINEAGE STATE: repair carrier commit `e259218b8091d883301fb53bdb19fd591a4bfdea` is queued as run `34792254835`. It is fail-before-publish: exact source/test replacements + focused/adversarial tests + py_compile + `git diff --check` must pass before it creates the Product repair commit. No duplicate run/PR is being created.

TRUTH UNCHANGED: this diagnosis grants zero canonical/training/unique-loss/exposure credit and does not authorize tokenizer fit, optimizer updates, training, learned weights or final-test access.

**Comment by Oleksii-debug** [comment]
SCHEMA_REPAIR CHECKPOINT — same-lineage PR910 Product repair is now physical and tested.

- repair carrier run `34792405618` / job `103818962761` terminal SUCCESS;
- focused/adversarial suite: **21 passed in 0.14s**; py_compile, diff-check and <=100-char source/test audit passed;
- tested Product commit published by the carrier: `51935f3e9942e2dd77c12268fabdd90a1126b9ec` (`D03: bind EDRNPA extraction to observed richtext schema`), changing only the incumbent tool + focused tests;
- v4 extraction is bound to exact observed `rna/database/document/item/richtext/par`, exact observed `item@name` / `par@def`, exactly one richtext item per document, and no longer consumes sibling metadata `<text>` as payload;
- all pre-existing bounded ZIP/XML/DTD/control-byte/size/UA/PII/dedup/deterministic-selection/zero-credit gates remain in the same lineage.

Immediately before the next Product mutation, PR910 was still open/non-draft/mergeable, main remained `2da11c62f066671a1122888ec116242502f4fb9c`, exact-key search still showed #1759 as the earlier valid claimant and #1763 as the later duplicate, and no PR910 branch run was in progress.

Real two-clean execution carrier is now exact head `37de5af9d983961a324b3c49b2903156a69e1145`, run `34792525578`. It performs two independent official downloads into fresh temp roots, two full streaming materializations, exact source/report/inventory/payload-identity comparison, and prints only whitelisted text-free aggregate evidence. Repository writes are disabled in this run. No source/candidate bytes are credited in advance; all canonical/training/optimized-target authority remains zero/false pending physical result and downstream gates.
